utils/user_info.py

```diff
@@ -144,18 +126,18 @@ def get_user_statistics(user):
     submissions = Submission.objects.filter(user=user)
     for label in status_labels:
         statistics += [{
-            'label': label,
-            'value': submissions.filter(status=label).count()
-        }]
+                           'label': label,
+                           'value': submissions.filter(status=label).count()
+                       }]
```

Comment:
prefer original indent, but you can discusse which one is better for your usage.